lib/stat_tracker.rb

```diff
@@ -12,110 +12,108 @@ def self.from_csv(file_paths)
   end
 
   def initialize(game_path, team_path, game_teams_path)
-    @game_path = game_path
-    @team_path = team_path
-    @game_teams_path = game_teams_path
+    @games = Games.new(game_path)
+    @teams = Teams.new(team_path)
+    @game_teams = GameTeams.new(game_teams_path)
   end
 
-  def games
-    Games.new(@game_path)
+#dont want a path as an instance varaible. ONly use path once
+#every time we call teams its initializing a new team every time
+  # def games
+  #   Games.new(@game_path)
+  # end
+  #
+  # def teams
+  #   Teams.new(@team_path)
+  # end
+  #
+  # def game_teams
+  #   GameTeams.new(@game_teams_path)
+  # end
+
+
+
+  def count_of_teams
+    @teams.all.size
+  end
+
+  # Name of the team with the highest average
+  # score per game across all seasons when they are away.
+  # {team_name: num_goals, team_name2: num_goals...}
+
+  def highest_scoring_visitor
```

Comment:
Ill go over whats going on in these methods when we get together tomorrow. THEY SUCK, but duh, of course, and duh of course of course a lot of similarities...